Blizzard.eai

```diff
@@ -587,24 +587,19 @@ function ZoomUp takes nothing returns nothing
 endfunction
 
 function InitZoom takes nothing returns nothing
-  local integer i = 0
-  if ZoomSetx == null then
-    set ZoomSetx = CreateTrigger()
-    set AdjustZoomUp = CreateTrigger()
-    set AdjustZoomDown = CreateTrigger()
-    call TriggerAddAction(ZoomSetx, function ZoomSet)
-    //call TriggerAddAction(AdjustZoomUp , function ZoomUp)
-    //call TriggerAddAction(AdjustZoomDown , function ZoomDown)
-    //loop
-    //  exitwhen i >= GetBJMaxPlayers()
-      //call SetCameraFieldForPlayer( Player(i), CAMERA_FIELD_TARGET_DISTANCE, 2500, 0 ) // not required in latest war3
-      if (GetPlayerController(GetLocalPlayer()) != MAP_CONTROL_COMPUTER) or IsPlayerObserver(GetLocalPlayer()) then
-        call TriggerRegisterPlayerChatEvent( ZoomSetx, GetLocalPlayer(), "-zoom", false )
-        //call TriggerRegisterPlayerKeyEventBJ( AdjustZoomUp, Player(i), bj_KEYEVENTTYPE_DEPRESS, bj_KEYEVENTKEY_UP ) // not required in latest war3
-        //call TriggerRegisterPlayerKeyEventBJ( AdjustZoomDown, Player(i), bj_KEYEVENTTYPE_DEPRESS, bj_KEYEVENTKEY_DOWN )
-      endif
-    //  set i = i + 1
-    //endloop
+  set ZoomSetx = CreateTrigger()
+  set AdjustZoomUp = CreateTrigger()
+  set AdjustZoomDown = CreateTrigger()
+  call TriggerAddAction(ZoomSetx, function ZoomSet)
+  call TriggerAddAction(AdjustZoomUp , function ZoomUp)
```

Comment:
These are not needed, this is all implemented officially by blizzard using pgup and pgdn keys.